notes_app/notes/tests.py

```diff
@@ -18,10 +18,15 @@ def setUp(self):
         self.api_detail = '/api/v1/note/1/'
         self.api_detail_shared = '/api/v1/note/4/'
         self.api_detail_not_shared = '/api/v1/note/5/'
+        self.note_keys = [
+            u'id', u'title', u'content', u'created_on', u'updated_on',
+            u'resource_uri', u'owner', u'shared_with'
+        ]
+        self.api_user_id = "/api/v1/user/2/"
         self.post_data = {
-            'title': 'New note title',
-            'content': 'Some content',
-            'owner': self.user.id
+            u'title': u'New note title',
```

Comment:
check importing of `unicode_literals` http://python-future.org/unicode_literals.html